src/main/java/com/kylenicholls/stash/parameterizedbuilds/ParameterizedBuildHook.java

```diff
@@ -133,6 +134,28 @@ public void validate(Settings settings, SettingsValidationErrors errors, Scope s
                 errors.addFieldError(SettingsService.PATH_PREFIX + i, pathException
                         .getDescription());
             }
+
+            PatternSyntaxException ignoreCommittersException = null;
```

Comment:
Since the Ignore Committers field is not a regex, we should omit this check. Otherwise valid usernames like `)user` will cause an error to be thrown.